paasta_tools/utils.py

```diff
@@ -135,6 +135,7 @@
     "kafkacluster",
     "monkrelays",
     "nrtsearchservice",
+    "spark",
```

Comment:
we should probably add a comment here since spark isn't really an instance type in the normal paasta sense (i.e., we don't have spark-$CLUSTER.yaml files in soaconfigs)